api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md

```diff
@@ -51,6 +51,172 @@ Accepts one of the following values.
+            // Change chart properties here
+            return chartOptions; // This line is optional
+        }
+        
+    The **chartOptions** object has the following structure:
+
+    <table class="dx-table full-width">
+        <tr>
+            <th>Field name</th>
+            <th>Type</th>
+            <th>Description</th>
+        </tr>
+        <tr>
+            <td><b>valueAxis</b></td>
+            <td>Array</td>
+            <td>Allows you to customize the automatically-created value axes. Each object in this array can contain fields described in the <a href="/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/" target="_blank">valueAxis</a> property description. When this array holds several objects, use their <a href="/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/#name" target="_blank">name</a> field to identify the axis.</td>
+        </tr>
+        <tr>
+            <td><b>panes</b></td>
+            <td>Array</td>
+            <td>Allows you to customize the automatically-created panes. Each object in this array can contain fields described in the <a href="/Documentation/ApiReference/UI_Components/dxChart/Configuration/panes/" target="_blank">panes</a> property description. When this array holds several objects, use their <a href="/Documentation/ApiReference/UI_Components/dxChart/Configuration/panes/#name" target="_blank">name</a> field to identify the pane.</td>
+        </tr>
+    </table>
+
+    You can also specify any other [Chart properties](/Documentation/ApiReference/UI_Components/dxChart/Configuration/) in this object, except [dataSource](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#dataSource).
+
+- **customizeSeries** 
+
+        <!--JavaScript-->customizeSeries: function (seriesName, seriesOptions) {
+            // Change series properties here
+            return seriesOptions; // This line is optional
+        }
+
+    The **seriesName** argument is a string that allows you to identify a series. The **seriesOptions** argument is an object with the following structure:
+
+    <table class="dx-table full-width">
+        <tr>
+            <th>Field name</th>
+            <th>Type</th>
+            <th>Description</th>
+        </tr>
+        <tr>
+            <td><b>axis</b></td>
+            <td>String</td>
+            <td>The name of the axis to which the series belongs. Has a value only when the chart has multiple axes.</td>
+        </tr>
+        <tr>
+            <td><b>pane</b></td>
+            <td>String</td>
+            <td>The name of the pane the series occupies. Has a value only when the chart has multiple panes.</td>
+        </tr>
+    </table>
+
+    You can also specify any other [series properties](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) in this object, except [seriesTemplate](Documentation/ApiReference/UI_Components/dxChart/Configuration/seriesTemplate/).
```

Comment:
Imho it should be 

----

PivotGrid code expects that this function will return a [series properties](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) object

Reply:
except [seriesTemplate] - no, see https://github.com/DevExpress/DevExtreme/blob/21_2/js/ui/pivot_grid/ui.pivot_grid.chart_integration.js#L260

Reply:
See my reply to [this comment](https://github.com/DevExpress/devextreme-documentation/pull/2884/files#r713782923).

Reply:
Again, our code allows both 'return' and 'modify': seriesOptions = extend(seriesOptions, customizeSeries(seriesName, seriesOptions));
The text is up to you.

>The seriesName argument is a string that allows you to identify a series. The seriesOptions argument is an object with the following structure:

fine.

>You can also specify any other series properties in this object, except seriesTemplate.

https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/ - the passed object doesn't contain the 'seriesTemplate' member.
This note of the 'customizeSeries' callback is not correct for the 'seriesOptions' argument.